CMakeLists.txt

```diff
@@ -287,14 +298,14 @@ IF (BUILD_STATIC_LIBRARY)
     TARGET_LINK_LIBRARIES (${STATIC_LIBRARY_TARGET} "crypto")
     TARGET_LINK_LIBRARIES (${STATIC_LIBRARY_TARGET} "ssl")
     TARGET_LINK_LIBRARIES (${STATIC_LIBRARY_TARGET} "curl")
-    TARGET_LINK_LIBRARIES (${STATIC_LIBRARY_TARGET} "Boost::filesystem")
+    TARGET_LINK_LIBRARIES (${STATIC_LIBRARY_TARGET} "Boost::filesystem" "Boost::url" ${Boost_STACKTRACE_BACKTRACE_LIBRARY})
```

Comment:
Sorry if this is just due to my ignorance, but do we need to explicitly include the boost backtrace lib here since the stacktrace/backtrace is implemented in core?

Reply:
slightly anecdotal, but it seems to build for me if I remove these.

Reply:
Let me confirm this, but from my testing it was required to generate a proper stack trace.

Reply:
Okay I checked this and either we need to link with "dl" or with "${Boost_STACKTRACE_BACKTRACE_LIBRARY}"
(I'm guessing the boost stacktrace backtrace library includes dl)